src/form.js

```diff
@@ -29,6 +29,7 @@ export default class Form extends React.Component {
     const card = Object.assign({}, this.state)
     this.props.onSubmit(card)
     event.target.reset()
+    window.location.reload()
```

Comment:
I don't believe this is necessary. Let me know if there is a specific reason you should keep this line.